### PR TITLE
Disable installation of the "certs" service, as it's not used in sandbox

### PR DESCRIPTION
**Pull request by riccardomurri**
The `edx_sandbox.yml` playbook will try to install the "certs" service and fail
because it does not enable the `certs` site in NginX.

This patch disables installation of "certs" altogether.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @riccardomurri! I've created [OSPR-402](https://openedx.atlassian.net/browse/OSPR-402) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a [signed contributor agreement](http://code.edx.org/individual-contributor-agreement.pdf) or indicated your institutional affiliation and added yourself to the [AUTHORS](https://github.com/edx/configuration/blob/master/AUTHORS) file. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) file for more information.

File: playbooks/edx_sandbox.yml [hunk 1]
```diff
@@ -40,9 +40,11 @@
     - forum
     - { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
     - { role: "xqueue", update_users: True }
-    - role: ora
```
Comment (sarina) [inline playbooks/edx_sandbox.yml]: why did you remove this?

**Comment by sarina** [comment]
@riccardomurri - can you please take a look at your build output and address the failures? Also please do follow the bot's instructions to become a contributor. Thanks!

File: playbooks/edx_sandbox.yml (hunk 1 above)
Comment (riccardomurri) [inline playbooks/edx_sandbox.yml]: Typo -- wil fix immediately

**Comment by riccardomurri** [comment]
Added myself to AUTHORS in PR #1849

**Comment by riccardomurri** [comment]
Regarding the "contributors' agreement": since this is stuff I'm doing at work, I've asked my manager whether _I_ can sign it or if someone else's signature is needed.

**Comment by sarina** [comment]
@riccardomurri - By signing an individual contributor agreement, you are representing that you own any contributions you make to the platform and can legally grant the rights set forth in the agreement.  If another party owns your contributions (e.g., your school or employer), you should reach out to Jennifer (contact info in the contributor's agreement document) and work out signing an institutional agreement.

Unfortunately we cannot proceed with any of your PRs until we've got the contributor's agreement worked out, so please reach out to Jennifer ASAP.

**Comment by riccardomurri** [comment]
Branch rebased against master, and contributor's agreement signed and sent to Jennifer.

**Comment by sarina** [comment]
Great, we have your agreement! Handing off to the proper team for review.

**Comment by feanil** [comment]
@riccardomurri   Why not just add certs to the nginx list instead?  The goal of the playbook is to be fairly comprehensive so I don't think it makes sense to remove the certs role from it.  If it is not working correctly I'm happy to accept PRs to fix it or issues with info for debugging.

**Comment by riccardomurri** [comment]
@feanil Yes, that's certainly an option. If you want, I can submit a PR for that.  Note however, that we do not use certs at our site, so I have never tested (and cannot test) that they work in the "sandbox" setup.

**Comment by feanil** [comment]
I think that is a better approach.  I can bring up something to test it here for this so let's make that change instead.

**Comment by riccardomurri** [comment]
Do you want me to change the commit in this PR, or should I open a new one?

**Comment by feanil** [comment]
Let's update this one.

**Comment by riccardomurri** [comment]
Just pushed the updated version.  Note that the branch name is now at odds with its content.

**Comment by feanil** [comment]
@riccardomurri I've verified that this fix is good.  I've found some other issues around certs upload that I've added as a bug to our backlog, but this fix was needed as well to get this all working.

:+1:
